.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
@andreypotapovaspose ,

looks like this was included in PR by mistake

Reply:
@yury-knml,
Thank you for the note. Fixed.